src/globals.jl

```diff
@@ -55,6 +55,13 @@ end
 
 propertynames(::GlobalsType, private::Bool=false) = Vector{Symbol}(Globals.NamesGVars())
 
+function replace_global!(name::Symbol, val::Any)
+    n = GapObj(name)
+    Wrappers.MakeReadWriteGlobal(n)
+    setproperty!(Globals, name, val)
+    Wrappers.MakeReadOnlyGlobal(n)
+end
```

Comment:
The analogous function `ReplaceGapFunc` in Oscar.jl/gap/OscarInterface/gap/OscarInterface.gi` creates a global variable `Concatenation("_ORIG_", name)` if it does not yet exist. Perhaps do this also here?